packages/driver/src/cy/listeners.js

```diff
@@ -51,6 +52,17 @@ module.exports = {
 
     // set onerror global handler
     contentWindow.onerror = callbacks.onError
+    contentWindow.addEventListener('unhandledrejection', ({ reason }) => {
+      if (reason.stack) {
+        const details = $stackUtils.getSourceDetailsForFirstLine(reason.stack, config('projectRoot'))
+
+        callbacks.onError(`Uncaught ${String(reason)}`, details.fileUrl, details.line, details.column, reason)
+
+        return
+      }
+
+      callbacks.onError(`Uncaught ${String(reason)}`, '', 0, 0, reason)
```

Comment:
The `Uncaught...` part of the string mimics how regular errors are reported (at least in Chrome) but it probably should be removed altogether and only `String(reason)` should be kept. 

As we have not been able to extract error details for the caught error here fields like filename, column and line are "nullified". Seems like a reasonable approach. WDYT?

Reply:
Yep probably just use `String(reason)` here

This all looks great :+1: